_pages/404.html

```diff
@@ -8,6 +8,6 @@
   <div class='container'>
         <h1>Ooops!</h1>
         <span>404</span>
-        <p><a href="/docs/content">Content</a> not found!</p>
+        <p><a href="/docs/content">Content</a>Not found!</p>
```

Comment:
Since it is a sentence ('Content not found') it should be remain as it was
```suggestion
        <p><a href="/docs/content">Content</a> not found!</p>
```

Reply:
done !